### ISSUE TITLE
WARN  spectit.ui.rcp.log.LogListener - Error occurred reading the existing versions from GitHub during check for new version job.

### ISSUE THREAD

**Issue by Yangsheng93** [inspectIT/inspectIT#209]
hi,The following error affects work:
```
2016-12-21 10:18:56,119: 18625  [       Worker-2] WARN  spectit.ui.rcp.log.LogListener - Error occurred reading the existing versions from GitHub during check for new version job.
java.net.UnknownHostException: api.github.com: 域名解析暂时失败
	at java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.net.InetAddress$1.lookupAllHostAddr(InetAddress.java:901)
	at java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1295)
	at java.net.InetAddress.getAllByName0(InetAddress.java:1248)
	at java.net.InetAddress.getAllByName(InetAddress.java:1164)
	at java.net.InetAddress.getAllByName(InetAddress.java:1098)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.resolveHostname(DefaultClientConnectionOperator.java:242)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:130)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:561)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:415)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:754)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:732)
	at rocks.inspectit.ui.rcp.job.CheckNewVersionJob.run(Unknown Source)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
```
I cannot access the network in LAN.Tip the above error, want to confirm whether it affects inspect ？

**Comment by pbouillet** [comment]
Hi @Yangsheng93 , this warning should not affect inspectIT. It just wants to check against GitHub if there is a newer version available. Where do you see this? Is it affecting inspectIT for you that you cannot do something? Or did you just noticed this message without any problems?

**Comment by pbouillet** [comment]
Hi @Yangsheng93 ,

I created an internal ticket for this (https://inspectit-performance.atlassian.net/browse/INSPECTIT-2320 ) to not show this error anymore if the UI does not have internet connection. For me, the UI should be still completely usable. If this is not the case, please let me know!

**Comment by Yangsheng93** [comment]
3Q

**Comment by ivansenic** [comment]
Fixed by #297
